cpu/esp32/include/periph_cpu.h

```diff
@@ -517,6 +517,10 @@ typedef struct {
 /** Timer used for system time */
 #define TIMER_SYSTEM    TIMERG0.hw_timer[0]
 
+/**
+ * @brief   Prevent shared timer functions from being used
+ */
+#define PERIPH_TIMER_PROVIDES_SET
```

Comment:
Where did that come from?

Reply:
good question...

Reply:
I'll remove maybe its a left over from previous chagnes

Reply:
This change seem valid though...

Reply:
Should I split out?

Reply:
Was already fixed in master, will remote

Reply:
Nah, I see there is a `timer_set` indeed, should be fine.